Libraries/LinkingIOS/RCTLinkingManager.m

```diff
@@ -42,15 +42,19 @@ - (void)stopObserving
   return @[@"url"];
 }
 
++ (BOOL)application:(UIApplication *)app
+            openURL:(NSURL *)URL
+            options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
+    [self postNotificationWithURL:URL];
+    return YES;
```

Comment:
decided to leave in `return YES` to be distinctly clear that this class function is returning a `BOOL`